src/main/java/com/qaprosoft/carina/demo/utils/MobileContextUtils.java

```diff
@@ -45,13 +45,17 @@ public void switchMobileContext(View context) {
         if (!isContextPresent) {
             throw new NotFoundException("Desired context is not present");
         }
-        LOGGER.info("Switching to context : " + context.getView());
+        LOGGER.info("Switching to context : " + desiredContext);
         ((SupportsContextSwitching) driver).context(desiredContext);
     }
 
     public enum View {
         NATIVE("NATIVE_APP"),
-        WEB("WEBVIEW_");
+        WEB_CARINA("WEBVIEW_com.solvd.carinademoapplication"),
```

Comment:
use previous version WEB only, we don't need to use full names of WebViews because we already know that they starts from `WEBVIEW_`